api/specs/webserver/openapi-projects.yaml

```diff
@@ -444,6 +444,32 @@ paths:
         default:
           $ref: "#/components/responses/DefaultErrorResponse"
 
+  /projects/{project_id}/nodes/{node_id}/restart:
```

Comment:
this path is at the end really appending ``:restart`` with ``:``, right? You wrote it differently in the description of the PR

Reply:
yes it does, if you look in the file below `api/specs/webserver/openapi.yaml` you will see it. Here it is not supported to use column `:`.

Reply:
here you do not even need to put the paths with ``/`` but just some kind of using key. 
In any case, all this will dissappear soon